common/changes/@uifabric/dashboard/fonts_theme_2018-10-10-23-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/dashboard",
+      "comment": "updating dashboard package",
```

Comment:
Please fix these comments per the other comment, but also, please use proper casing since these go directly into release notes.

Reply:
Just updated these comments! I specified what changed for each component in these packages.